### PR TITLE
Fix swagger local entry implemenation to support multi-tenant scenario

### PR DESCRIPTION
**Pull request by ChamodDamitha**
Related to [https://github.com/wso2/carbon-apimgt/pull/6128](https://github.com/wso2/carbon-apimgt/pull/6128)

### CODE REVIEW

File: components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/utils/LocalEntryAdminClient.java [hunk 1]
```diff
@@ -113,7 +114,7 @@ public void addLocalEntry(String content) throws AxisFault {
      */
     public Boolean getEntry(String key) throws AxisFault {
         try {
-            Object object = localEntryAdminServiceStub.getEntry(key);
+            Object object = localEntryAdminServiceStub.getEntry(key, tenantDomain);
```
Comment (fazlan-nazeem) [inline components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/utils/LocalEntryAdminClient.java]: can you name this variable appropriately?

File: components/apimgt/org.wso2.carbon.apimgt.gateway/src/main/java/org/wso2/carbon/apimgt/gateway/utils/LocalEntryClient.java [hunk 2]
```diff
@@ -67,7 +74,11 @@ public Boolean addLocalEntry(String content) throws AxisFault {
     public Object getEntry(String key) throws AxisFault {
         Object object;
         try {
-            object = localEntryAdminServiceStub.getEntry(key);
+            if (MultitenantConstants.SUPER_TENANT_DOMAIN_NAME.equals(tenantDomain)) {
+                object = localEntryAdminServiceStub.getEntry(key);
```
Comment (fazlan-nazeem) [inline components/apimgt/org.wso2.carbon.apimgt.gateway/src/main/java/org/wso2/carbon/apimgt/gateway/utils/LocalEntryClient.java]: change variable name

File: components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/utils/LocalEntryAdminClient.java (hunk 1 above)
Comment (ChamodDamitha) [reply components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/utils/LocalEntryAdminClient.java]: Updated

File: components/apimgt/org.wso2.carbon.apimgt.gateway/src/main/java/org/wso2/carbon/apimgt/gateway/utils/LocalEntryClient.java (hunk 2 above)
Comment (ChamodDamitha) [reply components/apimgt/org.wso2.carbon.apimgt.gateway/src/main/java/org/wso2/carbon/apimgt/gateway/utils/LocalEntryClient.java]: Updated

File: components/apimgt/org.wso2.carbon.apimgt.gateway/src/main/java/org/wso2/carbon/apimgt/gateway/utils/LocalEntryClient.java [hunk 3]
```diff
@@ -32,8 +33,10 @@
     private LocalEntryAdminServiceStub localEntryAdminServiceStub;
 
     static final String backendURLl = "local:///services/";
+    private String tenantDomain;
 
-    public LocalEntryClient() throws AxisFault {
+    public LocalEntryClient(String tenantDomain) throws AxisFault {
+        this.tenantDomain = tenantDomain;
         localEntryAdminServiceStub = new LocalEntryAdminServiceStub(null,
```
Comment (fazlan-nazeem) [inline components/apimgt/org.wso2.carbon.apimgt.gateway/src/main/java/org/wso2/carbon/apimgt/gateway/utils/LocalEntryClient.java]: When initialising the stub, better to pass the axis2 configuration context without passing null.

check https://github.com/wso2/carbon-apimgt/blob/6.x/components/apimgt/org.wso2.carbon.apimgt.keymgt.client/src/main/java/org/wso2/carbon/apimgt/keymgt/client/SubscriberKeyMgtClient.java#L48

Comment (ChamodDamitha) [reply components/apimgt/org.wso2.carbon.apimgt.gateway/src/main/java/org/wso2/carbon/apimgt/gateway/utils/LocalEntryClient.java]: Fixed

**Pull request merged**